src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.Netcore.csproj

```diff
@@ -24,6 +24,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net472'">
+    <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.5.0-preview" />
```

Comment:
It looks l9ike this isn't published.  Can we publish it?  Also, does this have the correct newtonsoft.json dependency in netstandard2.0?

Reply:
Package has been published and verified that it has proper `Newtonsoft.Json` dependency in .NET Standard